src/lib/ValidatorUtils/index.ts

```diff
@@ -0,0 +1,25 @@
+import { Validator } from 'types/validator';
+import { defaultProfilePicture } from 'assets/images';
+
+/**
+ * Returns the avatar picture associated to the provided validator.
+ * @param validator {Validator} which avatar picture should be returned.
+ */
+export const getValidatorAvatar = (validator: Validator) => {
+  if (
+    validator.profile?.profilePicture !== undefined &&
+    validator.profile.profilePicture.length > 0
+  ) {
```

Comment:
I think this should be possible replace it with the more intuitive `validatori?.profile?.profilePicture?.length > 0`. Since `undefined ≯ 0`, it should work